utils/plugin_requirements.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package utils
+
+import "github.com/mattermost/mattermost-server/model"
+
+func CheckRequiredConfig(requiredConfig, actualConfig *model.Config) (bool, error) {
```

Comment:
nit: I'd recommend locating this in the `plugin` package instead of having plugins depend on the ever changing `utils` directory. Although it's not strictly plugin specific, I suspect it's more clearly attached to our use case than general system handling.